app/assets/javascripts/game_moves.js

```diff
@@ -0,0 +1,27 @@
+$(function() {
+  $( ".piece_image" ).draggable();
+  $( "td" ).droppable({
+    
+    drop: function( event, ui ) {
+     var game_id = $('#board').data('game_id');
+
+      $.ajax( {  type: 'PUT',
+        url: '/games/' + game_id + '/move',
+        dataType: 'json',
+

```

Comment:
oh nice, a record of the moves! this'll be great if we want to put move logs in the ui anywhere.